airbyte-ci/connectors/metadata_service/lib/metadata_service/commands.py

```diff
@@ -1,7 +1,12 @@
-import click
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
```

Comment:
New guy question: What is the command that adds this?

Reply:
It's the pre-commit hook that does it.
`pre-commit install` and you should be good to go.